src/opers.c

```diff
@@ -2045,6 +2045,116 @@ CacheMethod(Obj oper, UInt n, Int prec, Obj * ids, Obj method)
     CHANGED_BAG(cacheBag);
 }
 
+static Obj ReturnTrue;
+static Obj VMETHOD_PRINT_INFO;
+
+// This function searches through the methods of operation <oper> with
+// arity <n>, looking for those matching the given <types>. Among these,
+// the <prec>-th is selected (<prec> starts at 0).
+//
+// If <verbose> is non-zero, the matching method are printed by calling
+// 'VMETHOD_PRINT_INFO'.
+//
+// If <constructor> is non-zero, then <oper> is a constructor, leading
+// to <types[0]> being treated differently.
+static ALWAYS_INLINE Obj GetMethodUncached_NEW(
```

Comment:
Maybe add a comment to explain the point of the ALWAYS_INLINE and that you expect it to be called with
some of the parameters compile-time constants.

Reply:
Will do.